packages/cactus-test-tooling/src/test/typescript/integration/corda/corda-test-ledger/corda-test-ledger-constructor.test.ts

```diff
@@ -1,7 +1,7 @@
 import test, { Test } from "tape-promise/tape";
 import { CordaTestLedger } from "../../../../../main/typescript/public-api";
 
-test("constructor throws if invalid input is provided", (assert: any) => {
+test("constructor throws if invalid input is provided", (assert: unknown) => {
```

Comment:
@yayoiukai2021 You can get rid of the compilation errors here with something like this:

```suggestion
test("constructor throws if invalid input is provided", (assert: Test) => {
```